force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -227,19 +283,21 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Book {
         // Declare three public instance variables - title, author, pages
-        // title
-        // author
-        // pages
+        public String title; 
+        public String author;
+        public Integer pages;
     
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(String newTitle, String newAuthor) {
             // Initialize the title and author
+            this.title = newTitle;
+            this.author = newAuthor;
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
             // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            return 'Title: ' +title + ', Author: ' +author;
```

Comment:
Although not required in Apex (unlike JavaScript), it's best to use the `this` keyword when referencing instance variables:
```suggestion
            return 'Title: ' + this.title + ', Author: ' + this.author;
```